addons/featured-dangos/addon.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Featured Dangos (April Fools 2023)",
```

Comment:
I'm pretty sure the plural for dango is the same as the singular.

Reply:
It's hard to know because it's a foreign word. I've seen the plural "dangos" in many places. There's thousands of results from Reddit posts: https://www.google.com/search?q=%22dangos%22+site:www.reddit.com

Reply:
It's like "emoji"/"emojis".

Either is fine with me, then.